src/Backend/Modules/MediaGalleries/Installer/Installer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Backend\Modules\MediaGalleries\Installer;
+
+use Backend\Core\Engine\Model;
+use Backend\Core\Installer\ModuleInstaller;
+use Backend\Modules\MediaGalleries\Domain\MediaGallery\MediaGallery;
+
+/**
+ * Installer for the media module
+ */
+class Installer extends ModuleInstaller
+{
+    /**
+     * Install the module
+     */
+    public function install()
+    {
+        $this->addModule('MediaGalleries');
```

Comment:
Fixed in https://github.com/forkcms/forkcms/pull/1986/commits/8aa2af5cfea6ddb214b831dad7ec997961d8852e